topics/transcriptomics/tutorials/goenrichment/tutorial.md

```diff
@@ -0,0 +1,343 @@
+> 
+**Figure 1** QuickGO - http://www.ebi.ac.uk/QuickGO
+
+**What are GO annotations?** <br/>
+Genes are associated to GO terms via GO annotations. Each gene can have multiple annotations, even of the same GO type. An important notion to take into account when using GO is that, according to the **true path rule**, a gene annotated to a term is also implicitly annotated to each ancestor of that term in the GO graph. GO annotations have evidence codes that encode the type of evidence supporting them: only a small minority of genes have experimentally verified  annotations; the large majority have annotations inferred electronically based on sequence homology or known patterns.
+
+> ### Overview
+>
+> In this tutorial, we will deal with:
+> 
+> 1. Functional Enrichment Analysis
+> 2. Methods to simplify the results
+> 3. Interpretation of the results
+> {:toc}
+>
+{: .agenda}
+
+ 
+# Functional Enrichment Analysis
+
+To perform functional enrichment analysis, we need to have:
+- A set of genes of interest (e.g., differentially expressed genes): **study set**
+- A set with all the genes to consider in the analysis: **population set** (which must contain the study set)
+- GO annotations, associating the genes in the population set to GO terms
+- The GO ontology, with the description of GO terms and their relationships
+ 
+For each GO term, we need to count the frequency (**k**) of genes in the study set (**n**) that are associated to the term, and the frequency (**K**) of genes in the population set (**N**) that are associated to the same term. Then we test how likely would it be to obtain at least **k** genes associated to the term if **n** genes would be randomly sampled from the population, given the frequency **K** and size **N** of the population.
+
+The appropriate statistical test is the one-tailed variant of Fisher’s exact test, also known as the hypergeometric test for over-representation. When the one-tailed version is applied, this test will compute the probability of observing at least the sample frequency, given the population frequency.  The [hypergeometric distribution](https://en.wikipedia.org/wiki/Hypergeometric_distribution) measures precisely the probability of **k** successes in **n** draws, without replacement, from a finite population of size **N** that contains exactly **K** successful objects: 
+> 
+![](../../images/goenrichment_formula.png)
+
+For this first exercise we will use data from [Trapnell et al. 2014](https://www.ncbi.nlm.nih.gov/pubmed/22383036 "Trapnell et al. data"). In this work, the authors created an artificial dataset of gene expression in *Drosophila melanogaster*, where 300 random genes were set (insilico) to be differentially expressed between two conditions.
+
+> ### {% icon hands_on %} Hands-on:
+> The data for this tutorial is available at [Zenodo](https://zenodo.org/record/1255038#.Wx4qTBwh3CI) to download.
+> 1. **Create a new history** 
+>
+> 2. **Upload to the Galaxy** the following files:
```

Comment:
Could maybe add the snippet for importing files from links?
 {% include snippets/import_via_link.md %}

Reply:
Thanks!

Yes, that would be good, but in this case the Zenodo link is a zip with some internal folders... when I upload this in Galaxy, it only gets the last file in the zip. I should probably make another Zenodo repository with a flat structure, and then I could import via link.

The files are relatively small in any case, and it's probably nice that the user can see them and explore them in their local computer...